src/accordion/AccordionItem.ts

```diff
@@ -19,20 +19,19 @@ export const useAccordionItem = createHook<
   AccordionItemHTMLProps
 >({
   name: "AccordionItem",
-  compose: [unstable_useId],
+  compose: unstable_useId,
   keys: ACCORDION_KEYS,
 
   useProps(options, { ref: htmlRef, ...htmlProps }) {
+    const { id, registerItem } = options;
```

Comment:
I was just following the reakit codebase style, I have never seen any destructuring in options.

So I did not either.

I will just destructure the state options and leave it's own component options.

What do you think, we can do, destructure of directly in the arguments itself?

Reply:
Well i think for larger options lists we should opt for destructuring on the function body instead of argument list directly.

and for smaller option list we can just directly destructure everything.

Reply:
Okay, acceptable. we both can follow the same. Thanks !!